assets/js/renderer.js

```diff
@@ -355,8 +355,51 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Clear any saved splitter-related div widths from local storage when the app closes.
+    const clearDivs = () => {
+        if ('localStorage' in window && window['localStorage'] !== null) {
+            localStorage.removeItem('splitSizes');
+        }
+    }
+
+    window.addEventListener('beforeunload', function (event) {
```

Comment:
Maybe we can use `sessionStorage` instead of `localStorage`? It looks like `sessionStorage` clears data automatically when the app is closed.